core/domain/topic_domain.py

```diff
@@ -779,9 +781,18 @@ def to_dict(self):
 class TopicRights(object):
     """Domain object for topic rights."""
 
-    def __init__(self, topic_id, manager_ids):
+    def __init__(self, topic_id, manager_ids, topic_is_published):
+        """Constructs a TopicRights domain object.
+
+        Args:
+            topic_id: str. The id of the topic.
+            manager_ids: list(str). The id of the users who have been assigned
+                as managers for the topic.
+            topic_is_published: boo. Whether the topic is playable by a learner.
```

Comment:
bool

Also, I'm not sure that a learner "plays" topics. Perhaps playable --> viewable.

Reply:
Done.